src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java

```diff
@@ -37,6 +37,9 @@
 @AGRCurationSchemaVersion(min = "1.2.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { ExperimentalCondition.class })
 public class ConditionRelation extends UniqueIdAuditedObject {
 
+	public static final String STANDARD = "Standard";
+	public static final String GENERIC_CONTROL = "Generic-control";
```

Comment:
Can these go into the constants class?

Reply:
I prefer having those constants on the class they belong to.

Reply:
Are they used outside the class at all? If not change them to private.

Reply:
I put them into an inner class of ConditionRelation. Constant which then makes it obvious what this constant is for and that it is a constant.